prompts/2024/english/prompt_2024_01/AnaColon_summary_Bingham_and_Ratcliff_2024

```diff
@@ -0,0 +1,17 @@
+Complex multicellularity has evolved five separate times in eukaryotes but has never evolved in prokaryotes. Complex multicellularity has been defined as a large organism composed of many cells that lives for an extensive period of time. Some prokaryotes are multicellular but they do not have the morphologies of complex multicellular organisms. 
+
+This paper proposes a new hypothesis as to why prokaryotes lack complex multicellularity compared to eukaryotes. When bottle neck events causing a reduced efficient population size cells are divided into groups. In response to this eukaryotes will undergo genomic expansion while prokaryotes will undergo genomic deletion. These differences in responses could be an explanation as to why eukaryotes have been able to evolve complex multicellularity and prokaryotes have not. 
```

Comment:
'When bottle neck events causing a reduced efficient population size cells are divided into groups.'
->
'When cells are divided into groups, this causes bottleneck events, with reduced *effective* population sizes.'